detect_secrets/plugins/private_key.py

```diff
@@ -39,12 +44,13 @@ class PrivateKeyDetector(RegexBasedDetector):
     """
 
     secret_type = 'Private Key'
+    MAX_FILE_SIZE: int = 4 * 1024
```

Comment:
@maxamel Where is this const used?

Reply:
@nimrodkor  right, I wanted to add file size limitation before reading file, I added it now, please re-review